ada/testsuite/tests/name_resolution/text_io_nested_pkgs/test.adb

```diff
@@ -0,0 +1,9 @@
+with Ada.Text_IO;
+
+procedure Test is
+   type U is mod 10;
+   package MIO is new Ada.Text_IO.Modular_IO (U);
```

Comment:
Have you checked that there isn't any other package that should be handled by this special mechanism ?

Reply:
Yes, `Modular_IO` specifically was missing